queue_and_stack/dll_stack.py

```diff
@@ -1,18 +1,29 @@
 import sys
-sys.path.append('../doubly_linked_list')
+sys.path.append('doubly_linked_list')
 from doubly_linked_list import DoublyLinkedList
 
 class Stack:
     def __init__(self):
         self.size = 0
         # Why is our DLL a good choice to store our elements?
-        # self.storage = ?
+        self.storage = DoublyLinkedList()
 
+    # adding so set the size to += 1
     def push(self, value):
-        pass
+        # pass
+        self.size += 1
+        return self.storage.add_to_head(value)
 
+    # removing so set the size -= 1
     def pop(self):
-        pass
```

Comment:
works but could do 
if self.size > 0:
    return self.storage.remove_from_head()
just less code

Reply:
thank you, I will try it